modAionImpl/src/org/aion/zero/impl/cli/Arguments.java

```diff
@@ -167,6 +167,28 @@
                     "drops all databases except for block and index when not given a parameter or starting from 0 and redoes import of all known main chain blocks")
     private String redoImport = null;
 
+    // data query
+    @Option(
+        names = {"qb", "-q block", "--query block"},
+        arity = "1",
+        paramLabel = "<block_number>",
+        description = "retrieve block information")
+    private String blockDetails = null;
+
+    @Option(
+        names = {"qt", "-q tx", "--query tx"},
+        arity = "1",
+        paramLabel = "<transaction hash>",
```

Comment:
Inconsistent `paramLabel` format. It should be `<transaction_hash>` and `<account_address>`.

Reply:
revised.